projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli_impl.py

```diff
@@ -31,14 +30,16 @@ class JobDeploy:
     ZIP_ARCHIVE_TYPE = "zip"
     ARCHIVE_SUFFIX = "-archive"
 
-    def __init__(self, rest_api_url: str, output):
+    def __init__(self, rest_api_url: str, output: str):
         self.deploy_api = ApiClientFactory(rest_api_url).get_deploy_api()
         self.jobs_api = ApiClientFactory(rest_api_url).get_jobs_api()
         self.job_sources_api = ApiClientFactory(rest_api_url).get_jobs_sources_api()
         # support for multiple deployments is not implemented yet so we can put anything here.
         # Ultimately this will be user facing parameter (possibly fetched from config.ini)
         self.__deployment_id = "production"
         self.__job_archive = JobArchive()
+        self.__output = output
```

Comment:
`self.__output` just holds the output format, right? If that's the case, it might be a good idea to rename it to `__output_format`. Furthermore, if it's not used for anything else, I suggest we just make it a class field of the printer instead.

Reply:
+1 for the renaming of "output"